predictive_models/baseline.py

```diff
@@ -0,0 +1,51 @@
+"""A Baseline model
+Source: https://www.tensorflow.org/tutorials/structured_data/time_series
+"""
+
+import tensorflow as tf
+import pandas as pd
+from window_generator import WindowGenerator
+
+
+# pylint: disable=too-few-public-methods
+class Baseline(tf.keras.Model):
+    def __init__(self, label_index=None):
+        super().__init__()
+        self.label_index = label_index
+
+    def call(self, inputs):
```

Comment:
Adding type hint will improve readability and helps avoiding trivial mistakes.
Let's make sure all inputs and outputs of all functions/methods are type-hinted.